### PR TITLE
Qt 6.4 support

### PR DESCRIPTION
**Pull request by inventron-tr**
This helps to migrate to Qt 6

### CODE REVIEW

File: scripts/linux/compile.sh [hunk 1]
```diff
@@ -38,13 +38,13 @@ fi
 if [ -d $parent_dir/output ]; then
 	read -p "Directory $parent_dir/output already exists. The script will delete and recreate it. Is that okay? (y/N) : " yn
 	case $yn in
-	    [Yy]* ) rm -rf $parent_dir/output ;;
+	    [Yy]* ) echo 1 ;;
```
Comment (pawelsalawa) [inline scripts/linux/compile.sh:41]: Should this remain in the PR? It goes against the logic of the script - user is asked if it's okay to delete the directory.

File: scripts/linux/compile.sh [hunk 2]
```diff
@@ -38,13 +38,13 @@ fi
...
 	    * ) echo "Aborted."; exit;;
 	esac
 fi
-
+#rm -rf $parent_dir/output
```
Comment (pawelsalawa) [inline scripts/linux/compile.sh:45]: Also a remaining of some work, that should not end up in the PR?

File: scripts/linux/compile.sh (hunk 1 above)
Comment (inventron-tr) [reply scripts/linux/compile.sh:41]: Sorry should remain as the original

File: scripts/linux/compile.sh (hunk 2 above)
Comment (inventron-tr) [reply scripts/linux/compile.sh]: yes will fix

**Comment by pawelsalawa** [comment]
So... this script switches from Qt5 to Qt6. I'm not sure if the sourcecode is already ready for Qt6. Have you tried it already?

**Comment by inventron-tr** [comment]
> So... this script switches from Qt5 to Qt6. I'm not sure if the sourcecode is already ready for Qt6. Have you tried it already?

Screens seems to be working right now. We are using it as an editor in our project. We didn't run any detailed tests

**Comment by inventron-tr** [comment]
> I'm not sure what the OP means by the screens which are working, but I had tried to build SQLiteStudio with Qt 6 a year or two ago, and have not succeeded. Still, I have these changes, and after some massaging, the main application builds and somewhat runs, although I can see something is wrong with fonts or their configuration. Nevertheless, this PR is definitely not enough.

Yes. During our project lifetime we will continue to fix the errors that we find. But currently ui is working for us. Table definition, updates, sql editor etc. Maybe this can stay at a branch for anyone who needs Qt 6.

**Pull request closed without merging**